swagger.json

```diff
@@ -0,0 +1,1035 @@
+{
+  "swagger": "2.0",
+  "info": {
+     "description": "A global Company that make travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.",
+     "version": "1.0.0",
+     "title": "firestar-backend",
```

Comment:
@ola357  why is this showing as your changes? Please pull `stage branch` to this branch to keep it up to date.

Reply:
I have. The branch is up to date